wrapper/src/main/java/software/amazon/jdbc/plugin/efm/MonitorServiceImpl.java

```diff
@@ -89,7 +94,12 @@ public MonitorConnectionContext startMonitoring(
           () -> Messages.get(
               "MonitorServiceImpl.emptyAliasSet",
               new Object[] {hostSpec}));
-      hostSpec.addAlias(hostSpec.asAlias());
+      try {
```

Comment:
I think we can remove this entire try-catch block. We just rely on provided nodeKeys, and if they empty then we raise an exception.